src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -2155,13 +2105,13 @@ public override string GetName(SyntaxNode declaration)
                 case SyntaxKind.MethodDeclaration:
                     return ((MethodDeclarationSyntax)declaration).Identifier.ValueText;
                 case SyntaxKind.FieldDeclaration:
-                    return GetName(((FieldDeclarationSyntax)declaration).Declaration);
+                    return this.GetName(((FieldDeclarationSyntax)declaration).Declaration);
```

Comment:
❔ Why are we adding `this.`? It's not consistent with the [style of the project](https://github.com/dotnet/roslyn/blob/master/.editorconfig#L31-L35). Therefore I would expect:

1. No new deviations from project style to be added
2. Corrections for existing deviations to be submitted as an isolated pull request that doesn't do anything else